pkg/controller/endpoint/endpoints_controller_test.go

```diff
@@ -1954,6 +1954,146 @@ func TestEndpointPortFromServicePort(t *testing.T) {
 	}
 }
 
+func TestMultipleServiceChanges(t *testing.T) {
+	mux := http.NewServeMux()
+	ns := metav1.NamespaceDefault
+	expectedSubsets := []v1.EndpointSubset{{
+		Addresses: []v1.EndpointAddress{
+			{IP: "1.2.3.4", NodeName: &emptyNodeName, TargetRef: &v1.ObjectReference{Kind: "Pod", Name: "pod0", Namespace: ns}},
+		},
+	}}
+	endpoint := &v1.Endpoints{
+		ObjectMeta: metav1.ObjectMeta{Name: "foo", Namespace: ns, ResourceVersion: "1"},
+		Subsets:    expectedSubsets,
+	}
+
+	var endpointsStore cache.Store
+	var endpoints *endpointController
+	blockDelete := make(chan struct{})
+	blockNextAction := make(chan struct{})
+	stopChan := make(chan struct{})
+
+	handlerFunc := func(res http.ResponseWriter, req *http.Request) {
+		if endpointsStore == nil {
+			res.WriteHeader(http.StatusInternalServerError)
+			res.Write([]byte("endpointsStore has not been set yet"))
+			return
+		}
+		if endpoints == nil {
+
```

Comment:
This feels like an unnecessary new line. On the other hand, a new line before the `if endpoints == nil` line would help match the rest of the structure here.